ray-operator/test/e2erayservice/rayservice_in_place_update_test.go

```diff
@@ -82,3 +83,187 @@ func TestRayServiceInPlaceUpdate(t *testing.T) {
 		g.Expect(stdout.String()).To(Equal("9 pizzas please!"))
 	}, TestTimeoutShort).Should(Succeed())
 }
+
+func TestRayServiceInPlaceUpdateWithRayClusterSpec(t *testing.T) {
```

Comment:
in-place update will not happen in this case.

```suggestion
func TestUpdateServeConfigAndRayClusterSpec(t *testing.T) {
```